### ISSUE TITLE
Implement new HTML theme

### ISSUE THREAD

**Issue by gingeleski** [ethereumsec/ethereumsec.github.io#2]
This issue constitutes a major UI overhaul to ideally be completed after #1

**Comment by gingeleski** [comment]
We'll basically call this *done* - it was in its own branch `issues_1+2` but has since been merged into `dev` branch.

There's some integration work to be done with the *course* content but that will be covered under **#3**.
